src/index.html

```diff
@@ -8,12 +8,6 @@
 <body>
     <header class="h-container">
         <nav class="nav-container">
-            <ul class="nav-items">
-                <li class="nav-item"><a href="#" class="nav-link">link 1</a></li>
-                <li class="nav-item"><a href="#" class="nav-link">link 2</a></li>
-                <li class="nav-item"><a href="#" class="nav-link">link 3</a></li>
-                <li class="nav-item"><a href="#" class="nav-link">link 4</a></li>
-            </ul>
```

Comment:
Please revert the links back, it's better to use them as HTML, instead of using CSS and pushing them into HTML. The operation is costly in Browsers